sys/include/phydat.h

```diff
@@ -98,6 +98,7 @@ enum {
     /* pressure */
     UNIT_BAR,       /**< Beer? */
     UNIT_PA,        /**< Pascal */
+    UNIT_HPA,       /**< hectoPascal */
```

Comment:
another thing is also: which would be our default unit for air pressure anyway? bmp180 and bmx280 (and my rework of mpl3115 #7120) use `UNIT_PA` and lps331ap uses `UNIT_BAR` (or mbar, with `scale = -3`).